### PR TITLE
Code Cleanup and Adding new Code !

### PR DESCRIPTION
**Pull request by akashsarkar188**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,17 +8,20 @@ android:
...
     - extra-android-m2repository
-
     # Specify at least one system image,
     # if you need to run emulator(s) during your tests
     - sys-img-x86-android-28
-#    - sys-img-armeabi-v7a-android-17
\ No newline at end of file
+#    - sys-img-armeabi-v7a-android-17
+
+ licenses:
```
Comment (mohitathwani) [inline .travis.yml]: What are these licenses for?

File: app/src/main/java/com/aspirebudgetingmobile/aspirebudgeting/activities/Login.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import com.google.android.gms.auth.api.signin.GoogleSignIn;
+import com.google.android.gms.auth.api.signin.GoogleSignInAccount;
+import com.google.android.gms.auth.api.signin.GoogleSignInClient;
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions;
+import com.google.android.gms.common.api.ApiException;
+import com.google.android.gms.common.api.Scope;
+import com.google.android.gms.tasks.Task;
+import com.google.android.material.card.MaterialCardView;
+
+public class Login extends AppCompatActivity {
```
Comment (mohitathwani) [inline app/src/main/java/com/aspirebudgetingmobile/aspirebudgeting/activities/Login.java]: Is this Login class a view that the user will see? If yes, can you abstract the interaction with the Google Sign In sdk away from this class and put it in it's own class? Refer to Facilities/UserManager.swift in iOS

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">aspirebudgeting</string>
-    <string name="title_activity_login">LoginActivity</string>
-    <string name="action_settings">Settings</string>
+    <string name="app_name">Aspire</string>
+
+    <string name="GOOGLE_CLIEND_ID">986830921910-r49kvh077ecbugterud2i0m0hati2e3v.apps.googleusercontent.com</string>
```
Comment (mohitathwani) [inline app/src/main/res/values/strings.xml]: It's not safe to commit this info to the repository. Let's get on a call and I'll explain how I hide it in iOS.

Comment (akashsarkar188) [reply app/src/main/res/values/strings.xml]: Actually, we can even work without the key, I just kept it for trial purpose. I will remove it.

File: .travis.yml (hunk 1 above)
Comment (akashsarkar188) [reply .travis.yml]: I was following the documentation of travis I am not yet sure if it is needed or not.

**Pull request merged**